### PR TITLE
README: blurb readme draft

### PR DESCRIPTION
**Pull request by friendlyantz**
Hey team,
See my README blurb / draft for your review. Might be worth while making a YouTube vieo to showcase the app. I put Square video as a placeholder.

Also, not sure if we need original code from readme though.
```bash
make
bin/setup

# add yourself an admin user and login with reset password link
bin/rails 'admin:make_admin_user[m@m.m]'

# enable|disable features
bin/rails 'admin:flipper[portal,enable]' IDENTITIES=MLWS55T9N0BQ9,MLKR1MSQCYV7Y
bin/rails 'admin:flipper[portal,disable]' IDENTITIES=MLWS55T9N0BQ9
```

image
image
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,48 @@
...
+You'll need the following installed to run the template successfully:
 
+* Ruby 3.0.1 or higher
+* bundler - `gem install bundler`
+* rails - `gem install rails`
+* Yarn - `brew install yarn` or [Install Yarn](https://yarnpkg.com/en/docs/install)
+
+Other:
+* Svelte Compiler/ JS framework (Provided out of the box) - lightweight and fast alternative to React/Vue/Angular
+* Database (MySQL Provided out of the box)
```
Comment (saramic) [inline README.md:17]: does it work with mysql? we have postgres and the assoicated gems so it should work with postgresql. Again I am not sure how to ensure this. in general I use `.tool-versions` which is part of asdf to manage all of that (you should too -> https://asdf-vm.com/#/core-manage-asdf) maybe we just mandate that to work on this project and get all dependencies right you need asdf

**Review by saramic — COMMENTED** [review]
hopefully some points worthy of discussion tonight :)_

File: README.md [hunk 2]
```diff
@@ -1,12 +1,48 @@
...
+```
+paste your key in the above file and make sure it stays private
+## Ruby Gems
+```zsh
+bundle install
 ```
+## Yarn
+```zsh
+yarn install
+```
```
[the remark this answers is not in the document]
Comment (friendlyantz) [reply README.md:34]: never done this before, will update accordingly

[the remark this answers is not in the document]
Comment (friendlyantz) [reply README.md:34]: tested, all ok on my machine. might me worthwhile looking at master key?
image

File: README.md [hunk 3]
```diff
@@ -1,12 +1,48 @@
...
 
-[![CircleCI](https://circleci.com/gh/failure-driven/square-snippet-demo.svg?style=svg)](https://circleci.com/gh/failure-driven/square-snippet-demo)
+## Requirements
 
-</div>
+You'll need the following installed to run the template successfully:
 
+* Ruby 3.0.1 or higher
+* bundler - `gem install bundler`
+* rails - `gem install rails`
```
[the remark this answers is not in the document]
Comment (friendlyantz) [reply README.md:12]: great. trust `bunle` is launched via make -> bin/setup?

File: README.md [hunk 4]
```diff
@@ -1,12 +1,48 @@
...
+* Yarn - `brew install yarn` or [Install Yarn](https://yarnpkg.com/en/docs/install)
```
[the remark this answers is not in the document]
Comment (friendlyantz) [reply README.md:13]: Noted. never heard of asdf. will update to `npm install --global yarn`

File: README.md [hunk 5]
```diff
@@ -1,12 +1,48 @@
-<h1 align="center">Square Snippet Demo - swif.club</h1>
+# SWIF - Social Shopping app powered by Square Unboxed - API snippet [![CircleCI](https://circleci.com/gh/failure-driven/square-snippet-demo.svg?style=svg)](https://circleci.com/gh/failure-driven/square-snippet-demo)
```
[the remark this answers is not in the document]
Comment (friendlyantz) [reply README.md:1]: done

File: README.md [hunk 6]
```diff
@@ -1,12 +1,48 @@
...
-</div>
+You'll need the following installed to run the template successfully:
 
+* Ruby 3.0.1 or higher
+* bundler - `gem install bundler`
+* rails - `gem install rails`
+* Yarn - `brew install yarn` or [Install Yarn](https://yarnpkg.com/en/docs/install)
+
+Other:
+* Svelte Compiler/ JS framework (Provided out of the box) - lightweight and fast alternative to React/Vue/Angular
```
[the remark this answers is not in the document]
Comment (friendlyantz) [reply README.md:16]: done

File: README.md [hunk 7]
```diff
@@ -1,12 +1,48 @@
...
+* Database (MySQL Provided out of the box)
+
+# Commands to run / Setting up
+## Set credentials key
+```zsh
+touch config/credentials/development.key
+echo '/config/credentials/development.key' >> .gitignore
+git status # '/config/credentials/development.key' should not be there, we don't want to push it to Github.
```
[the remark this answers is not in the document]
Comment (friendlyantz) [reply README.md:24]: done

File: README.md [hunk 8]
```diff
@@ -1,12 +1,48 @@
...
-<div align="center">
+[![Square Unboxed - API snippet for Rails Template Walkthrough](https://i.ibb.co/6X4wXtw/Screen-Shot-2021-06-08-at-2-28-36-pm.png)](https://www.youtube.com/watch?v=JbytzNibk54)
 
-[![CircleCI](https://circleci.com/gh/failure-driven/square-snippet-demo.svg?style=svg)](https://circleci.com/gh/failure-driven/square-snippet-demo)
+## Requirements
 
-</div>
+You'll need the following installed to run the template successfully:
 
+* Ruby 3.0.1 or higher
```
[the remark this answers is not in the document]
Comment (friendlyantz) [reply README.md]: lets look into it

**Pull request merged**